fc/jira/tasks.py

```diff
@@ -33,15 +33,27 @@ def score_triage_and_el_tasks(auth: Auth):
     if len(scoring_results_exception) > 0:
         cli_library.fail_execution(1, 'Task scoring failed')
 
+    scoring_results = [scoring_result for scoring_result in scoring_results if
```

Comment:
We probably don't need to overwrite `scoring_result` since at this point, if there was an `Exception`, `fcli` would have errored out.